src/index.js

```diff
@@ -31,3 +32,5 @@ yargs
 	.demandCommand()
 	.strict()
 	.argv;
+
+updateNotifier({pkg}).notify({ defer: true });
```

Comment:
This should be before all the `yargs(...)` stuff takes place -- aka, right after the `checkVersion()`.

Also, we don't need `defer:true` --- it's `true` by default.

Reply:
Done